tests/player.test.js

```diff
@@ -68,19 +68,21 @@ describe('Player:', function () {
   it('like and dislike a video as an anonymous user @watch', () => {
     assertUserIsNotLoggedIn(browser)
     browser.url('http://localhost:3000/play/12345?playlist=98765')
-    browser.pause(2000)
+    browser.pause(1000)
     browser.waitForClickable('#button-like')
     assert.equal(browser.getText('#button-like'), '')
     assert.equal(browser.getText('#button-dislike'), '')
 
     browser.click('#button-like')
-    browser.pause(2000)
+    browser.pause(1000)
     assert.equal(browser.getText('#button-like'), '1')
```

Comment:
For example, this could be rewritten as:

```
browser.waitUntil(() => {
  browser.getText('#button-like') === '1'
});
```

Reply:
ok